com.unity.multiplayer.mlapi/Runtime/Transports/NetworkTransport.cs

```diff
@@ -5,6 +5,11 @@
 
 namespace MLAPI.Transports
 {
+    public class DontShowInTransportDropdownAttribute : Attribute
+    {
+
+    }
```

Comment:
since this is going to target Editor inspector rendering, we might move this attribute to Editor asmdef/namespace as well, no?

Reply:
Then we have to cross reference them and I couldn't get it working